### ISSUE TITLE
Test fail on Ubuntu 16.04

### ISSUE THREAD

**Issue by adamryczkowski** [JuliaIO/ImageMagick.jl#135]
This is the traceback starting from fresh `.julia`.
```
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.0.0 (2018-08-08)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

julia> import Pkg

julia> Pkg.add("ImageMagick")
   Cloning default registries into /home/adam/.julia/registries
   Cloning registry General from "https://github.com/JuliaRegistries/General.git"
  Updating registry at `~/.julia/registries/General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
 Installed PaddedViews ─────── v0.4.1
 Installed BinaryProvider ──── v0.4.2
 Installed FixedPointNumbers ─ v0.5.2
 Installed Conda ───────────── v1.0.1
 Installed Compat ──────────── v1.1.0
 Installed ImageMagick ─────── v0.7.1
 Installed FileIO ──────────── v1.0.1
 Installed Colors ──────────── v0.9.4
 Installed NaNMath ─────────── v0.3.2
 Installed Graphics ────────── v0.3.0
 Installed OffsetArrays ────── v0.8.1
 Installed VersionParsing ──── v1.1.2
 Installed JSON ────────────── v0.19.0
 Installed ImageCore ───────── v0.7.1
 Installed FFTW ────────────── v0.2.4
 Installed Reexport ────────── v0.2.0
 Installed ColorTypes ──────── v0.7.4
 Installed MappedArrays ────── v0.2.1
 Installed AbstractFFTs ────── v0.3.2
  Updating `~/.julia/environments/v1.0/Project.toml`
  [6218d12a] + ImageMagick v0.7.1
  Updating `~/.julia/environments/v1.0/Manifest.toml`
  [621f4979] + AbstractFFTs v0.3.2
  [b99e7846] + BinaryProvider v0.4.2
  [3da002f7] + ColorTypes v0.7.4
  [5ae59095] + Colors v0.9.4
  [34da2185] + Compat v1.1.0
  [8f4d0f93] + Conda v1.0.1
  [7a1cc6ca] + FFTW v0.2.4
  [5789e2e9] + FileIO v1.0.1
  [53c48c17] + FixedPointNumbers v0.5.2
  [a2bd30eb] + Graphics v0.3.0
  [a09fc81d] + ImageCore v0.7.1
  [6218d12a] + ImageMagick v0.7.1
  [682c06a0] + JSON v0.19.0
  [dbb5928d] + MappedArrays v0.2.1
  [77ba4419] + NaNMath v0.3.2
  [6fe1bfb0] + OffsetArrays v0.8.1
  [5432bcbf] + PaddedViews v0.4.1
  [189a3867] + Reexport v0.2.0
  [81def892] + VersionParsing v1.1.2
  [2a0f44e3] + Base64 
  [ade2ca70] + Dates 
  [8bb1440f] + DelimitedFiles 
  [8ba89e20] + Distributed 
  [b77e0a4c] + InteractiveUtils 
  [76f85450] + LibGit2 
  [8f399da3] + Libdl 
  [37e2e46d] + LinearAlgebra 
  [56ddb016] + Logging 
  [d6f4376e] + Markdown 
  [a63ad114] + Mmap 
  [44cfe95a] + Pkg 
  [de0858da] + Printf 
  [3fa0cd96] + REPL 
  [9a3f8284] + Random 
  [ea8e919c] + SHA 
  [9e88b42a] + Serialization 
  [1a1011a3] + SharedArrays 
  [6462fe0b] + Sockets 
  [2f01184e] + SparseArrays 
  [10745b16] + Statistics 
  [8dfed614] + Test 
  [cf7118a7] + UUIDs 
  [4ec0a83e] + Unicode 
  Building Conda ──────→ `~/.julia/packages/Conda/m7vem/deps/build.log`
  Building FFTW ───────→ `~/.julia/packages/FFTW/p7sLQ/deps/build.log`
  Building ImageMagick → `~/.julia/packages/ImageMagick/d5KBL/deps/build.log`

julia> Pkg.test("ImageMagick")
   Testing ImageMagick
 Resolving package versions...
 Installed CoordinateTransformations ─ v0.5.0
 Installed ImageTransformations ────── v0.5.0
 Installed ZipFile ─────────────────── v0.7.0
 Installed MacroTools ──────────────── v0.4.4
 Installed ImageMetadata ───────────── v0.5.0
 Installed ColorVectorSpace ────────── v0.6.1
 Installed DataStructures ──────────── v0.11.1
 Installed IndirectArrays ──────────── v0.5.0
 Installed Ratios ──────────────────── v0.3.0
 Installed RangeArrays ─────────────── v0.3.1
 Installed SimpleTraits ────────────── v0.8.0
 Installed URIParser ───────────────── v0.4.0
 Installed IdentityRanges ──────────── v0.2.0
 Installed IterTools ───────────────── v1.0.0
 Installed AxisAlgorithms ──────────── v0.3.0
 Installed Interpolations ──────────── v0.8.0
 Installed Missings ────────────────── v0.3.0
 Installed ShowItLikeYouBuildIt ────── v0.2.0
 Installed ImageAxes ───────────────── v0.5.0
 Installed SortingAlgorithms ───────── v0.3.1
 Installed IntervalSets ────────────── v0.2.0
 Installed SpecialFunctions ────────── v0.7.0
 Installed AxisArrays ──────────────── v0.3.0
 Installed WoodburyMatrices ────────── v0.4.1
 Installed Rotations ───────────────── v0.8.0
 Installed BinDeps ─────────────────── v0.8.10
 Installed StaticArrays ────────────── v0.8.3
 Installed StatsBase ───────────────── v0.25.0
  Building SpecialFunctions → `~/.julia/packages/SpecialFunctions/KvXoO/deps/build.log`
    Status `/tmp/tmpQGHN9L/Manifest.toml`
  [621f4979] AbstractFFTs v0.3.2
  [13072b0f] AxisAlgorithms v0.3.0
  [39de3d68] AxisArrays v0.3.0
  [9e28174c] BinDeps v0.8.10
  [b99e7846] BinaryProvider v0.4.2
  [3da002f7] ColorTypes v0.7.4
  [c3611d14] ColorVectorSpace v0.6.1
  [5ae59095] Colors v0.9.4
  [34da2185] Compat v1.1.0
  [8f4d0f93] Conda v1.0.1
  [150eb455] CoordinateTransformations v0.5.0
  [864edb3b] DataStructures v0.11.1
  [7a1cc6ca] FFTW v0.2.4
  [5789e2e9] FileIO v1.0.1
  [53c48c17] FixedPointNumbers v0.5.2
  [a2bd30eb] Graphics v0.3.0
  [bbac6d45] IdentityRanges v0.2.0
  [2803e5a7] ImageAxes v0.5.0
  [a09fc81d] ImageCore v0.7.1
  [6218d12a] ImageMagick v0.7.1
  [bc367c6b] ImageMetadata v0.5.0
  [02fcd773] ImageTransformations v0.5.0
  [9b13fd28] IndirectArrays v0.5.0
  [a98d9a8b] Interpolations v0.8.0
  [8197267c] IntervalSets v0.2.0
  [c8e1da08] IterTools v1.0.0
  [682c06a0] JSON v0.19.0
  [1914dd2f] MacroTools v0.4.4
  [dbb5928d] MappedArrays v0.2.1
  [e1d29d7a] Missings v0.3.0
  [77ba4419] NaNMath v0.3.2
  [6fe1bfb0] OffsetArrays v0.8.1
  [5432bcbf] PaddedViews v0.4.1
  [b3c3ace0] RangeArrays v0.3.1
  [c84ed2f1] Ratios v0.3.0
  [189a3867] Reexport v0.2.0
  [6038ab10] Rotations v0.8.0
  [9966252f] ShowItLikeYouBuildIt v0.2.0
  [699a6c99] SimpleTraits v0.8.0
  [a2af1166] SortingAlgorithms v0.3.1
  [276daf66] SpecialFunctions v0.7.0
  [90137ffa] StaticArrays v0.8.3
  [2913bbd2] StatsBase v0.25.0
  [30578b45] URIParser v0.4.0
  [81def892] VersionParsing v1.1.2
  [efce3f68] WoodburyMatrices v0.4.1
  [a5390f91] ZipFile v0.7.0
  [2a0f44e3] Base64  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Base64`]
  [ade2ca70] Dates  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Dates`]
  [8bb1440f] DelimitedFiles  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/DelimitedFiles`]
  [8ba89e20] Distributed  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Distributed`]
  [b77e0a4c] InteractiveUtils  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/InteractiveUtils`]
  [76f85450] LibGit2  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/LibGit2`]
  [8f399da3] Libdl  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Libdl`]
  [37e2e46d] LinearAlgebra  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/LinearAlgebra`]
  [56ddb016] Logging  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Logging`]
  [d6f4376e] Markdown  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Markdown`]
  [a63ad114] Mmap  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Mmap`]
  [44cfe95a] Pkg  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Pkg`]
  [de0858da] Printf  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Printf`]
  [3fa0cd96] REPL  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/REPL`]
  [9a3f8284] Random  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Random`]
  [ea8e919c] SHA  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/SHA`]
  [9e88b42a] Serialization  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Serialization`]
  [1a1011a3] SharedArrays  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/SharedArrays`]
  [6462fe0b] Sockets  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Sockets`]
  [2f01184e] SparseArrays  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/SparseArrays`]
  [10745b16] Statistics  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Statistics`]
  [8dfed614] Test  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Test`]
  [cf7118a7] UUIDs  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/UUIDs`]
  [4ec0a83e] Unicode  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Unicode`]
WARNING: could not import Base.atan2 into ColorVectorSpace
WARNING: could not import LinearAlgebra.gradient into Interpolations
WARNING: could not import Base.start into ImageTransformations
WARNING: could not import Base.next into ImageTransformations
WARNING: could not import Base.done into ImageTransformations
[ Info: ("ImageMagick version ", v"6.9.10-10")
ERROR: LoadError: LoadError: ArgumentError: Package ImageShow not found in current path:
- Run `Pkg.add("ImageShow")` to install the ImageShow package.

Stacktrace:
 [1] require(::Module, ::Symbol) at ./loading.jl:817
 [2] include at ./boot.jl:317 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1038
 [4] include(::Module, ::String) at ./sysimg.jl:29
 [5] include(::String) at ./client.jl:388
 [6] top-level scope at none:0
 [7] include at ./boot.jl:317 [inlined]
 [8] include_relative(::Module, ::String) at ./loading.jl:1038
 [9] include(::Module, ::String) at ./sysimg.jl:29
 [10] include(::String) at ./client.jl:388
 [11] top-level scope at none:0
in expression starting at /home/adam/.julia/packages/ImageMagick/d5KBL/test/constructed_images.jl:2
in expression starting at /home/adam/.julia/packages/ImageMagick/d5KBL/test/runtests.jl:18
ERROR: Package ImageMagick errored during testing
Stacktrace:
 [1] pkgerror(::String, ::Vararg{String,N} where N) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Types.jl:120
 [2] macro expansion at ./logging.jl:313 [inlined]
 [3] #test#61(::Bool, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:1283
 [4] #test at ./none:0 [inlined]
 [5] #test#44(::Bool, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:256
 [6] test at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:242 [inlined]
 [7] #test#43 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:239 [inlined]
 [8] test at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:239 [inlined]
 [9] #test#42 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:238 [inlined]
 [10] test at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:238 [inlined]
 [11] #test#41 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:237 [inlined]
 [12] test(::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:237
 [13] top-level scope at none:0

julia> Pkg.add("ImageShow")
 Resolving package versions...
 Installed Requires ── v0.5.2
 Installed ImageShow ─ v0.1.0
  Updating `~/.julia/environments/v1.0/Project.toml`
  [4e3cecfd] + ImageShow v0.1.0
  Updating `~/.julia/environments/v1.0/Manifest.toml`
  [4e3cecfd] + ImageShow v0.1.0
  [ae029012] + Requires v0.5.2

julia> Pkg.test("ImageMagick")
   Testing ImageMagick
 Resolving package versions...
    Status `/tmp/tmp1kVIkE/Manifest.toml`
  [621f4979] AbstractFFTs v0.3.2
  [13072b0f] AxisAlgorithms v0.3.0
  [39de3d68] AxisArrays v0.3.0
  [9e28174c] BinDeps v0.8.10
  [b99e7846] BinaryProvider v0.4.2
  [3da002f7] ColorTypes v0.7.4
  [c3611d14] ColorVectorSpace v0.6.1
  [5ae59095] Colors v0.9.4
  [34da2185] Compat v1.1.0
  [8f4d0f93] Conda v1.0.1
  [150eb455] CoordinateTransformations v0.5.0
  [864edb3b] DataStructures v0.11.1
  [7a1cc6ca] FFTW v0.2.4
  [5789e2e9] FileIO v1.0.1
  [53c48c17] FixedPointNumbers v0.5.2
  [a2bd30eb] Graphics v0.3.0
  [bbac6d45] IdentityRanges v0.2.0
  [2803e5a7] ImageAxes v0.5.0
  [a09fc81d] ImageCore v0.7.1
  [6218d12a] ImageMagick v0.7.1
  [bc367c6b] ImageMetadata v0.5.0
  [02fcd773] ImageTransformations v0.5.0
  [9b13fd28] IndirectArrays v0.5.0
  [a98d9a8b] Interpolations v0.8.0
  [8197267c] IntervalSets v0.2.0
  [c8e1da08] IterTools v1.0.0
  [682c06a0] JSON v0.19.0
  [1914dd2f] MacroTools v0.4.4
  [dbb5928d] MappedArrays v0.2.1
  [e1d29d7a] Missings v0.3.0
  [77ba4419] NaNMath v0.3.2
  [6fe1bfb0] OffsetArrays v0.8.1
  [5432bcbf] PaddedViews v0.4.1
  [b3c3ace0] RangeArrays v0.3.1
  [c84ed2f1] Ratios v0.3.0
  [189a3867] Reexport v0.2.0
  [6038ab10] Rotations v0.8.0
  [9966252f] ShowItLikeYouBuildIt v0.2.0
  [699a6c99] SimpleTraits v0.8.0
  [a2af1166] SortingAlgorithms v0.3.1
  [276daf66] SpecialFunctions v0.7.0
  [90137ffa] StaticArrays v0.8.3
  [2913bbd2] StatsBase v0.25.0
  [30578b45] URIParser v0.4.0
  [81def892] VersionParsing v1.1.2
  [efce3f68] WoodburyMatrices v0.4.1
  [a5390f91] ZipFile v0.7.0
  [2a0f44e3] Base64  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Base64`]
  [ade2ca70] Dates  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Dates`]
  [8bb1440f] DelimitedFiles  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/DelimitedFiles`]
  [8ba89e20] Distributed  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Distributed`]
  [b77e0a4c] InteractiveUtils  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/InteractiveUtils`]
  [76f85450] LibGit2  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/LibGit2`]
  [8f399da3] Libdl  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Libdl`]
  [37e2e46d] LinearAlgebra  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/LinearAlgebra`]
  [56ddb016] Logging  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Logging`]
  [d6f4376e] Markdown  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Markdown`]
  [a63ad114] Mmap  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Mmap`]
  [44cfe95a] Pkg  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Pkg`]
  [de0858da] Printf  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Printf`]
  [3fa0cd96] REPL  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/REPL`]
  [9a3f8284] Random  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Random`]
  [ea8e919c] SHA  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/SHA`]
  [9e88b42a] Serialization  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Serialization`]
  [1a1011a3] SharedArrays  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/SharedArrays`]
  [6462fe0b] Sockets  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Sockets`]
  [2f01184e] SparseArrays  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/SparseArrays`]
  [10745b16] Statistics  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Statistics`]
  [8dfed614] Test  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Test`]
  [cf7118a7] UUIDs  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/UUIDs`]
  [4ec0a83e] Unicode  [`/home/Adama-docs/Adam/linux/tmp/julia-1.0.0/bin/../share/julia/stdlib/v1.0/Unicode`]
[ Info: ("ImageMagick version ", v"6.9.10-10")
ERROR: LoadError: LoadError: ArgumentError: Package ImageShow not found in current path:
- Run `Pkg.add("ImageShow")` to install the ImageShow package.

Stacktrace:
 [1] require(::Module, ::Symbol) at ./loading.jl:817
 [2] include at ./boot.jl:317 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1038
 [4] include(::Module, ::String) at ./sysimg.jl:29
 [5] include(::String) at ./client.jl:388
 [6] top-level scope at none:0
 [7] include at ./boot.jl:317 [inlined]
 [8] include_relative(::Module, ::String) at ./loading.jl:1038
 [9] include(::Module, ::String) at ./sysimg.jl:29
 [10] include(::String) at ./client.jl:388
 [11] top-level scope at none:0
in expression starting at /home/adam/.julia/packages/ImageMagick/d5KBL/test/constructed_images.jl:2
in expression starting at /home/adam/.julia/packages/ImageMagick/d5KBL/test/runtests.jl:18
ERROR: Package ImageMagick errored during testing
Stacktrace:
 [1] pkgerror(::String, ::Vararg{String,N} where N) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Types.jl:120
 [2] macro expansion at ./logging.jl:313 [inlined]
 [3] #test#61(::Bool, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/Operations.jl:1283
 [4] #test at ./none:0 [inlined]
 [5] #test#44(::Bool, ::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}, ::Function, ::Pkg.Types.Context, ::Array{Pkg.Types.PackageSpec,1}) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:256
 [6] test at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:242 [inlined]
 [7] #test#43 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:239 [inlined]
 [8] test at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:239 [inlined]
 [9] #test#42 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:238 [inlined]
 [10] test at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:238 [inlined]
 [11] #test#41 at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:237 [inlined]
 [12] test(::String) at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.0/Pkg/src/API.jl:237
 [13] top-level scope at none:0

julia> 
```

```
Julia Version 1.0.0
Commit 5d4eaca0c9 (2018-08-08 20:58 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.0 (ORCJIT, skylake)```

**Comment by SimonDanisch** [comment]
that should be fine and is a consequence of merging a half backed solution!
@RalphAS, any chance we can turn this into a full solution soon? Just a tagging ImageShow should already improve the situation quite a bit, no?

**Comment by timholy** [comment]
Fixed by #136
